src/screen.c

```diff
@@ -4738,6 +4740,7 @@ static struct charstab filltab[] =
     CHARSTAB_ENTRY(&fill_chars.foldopen,    "foldopen"),
     CHARSTAB_ENTRY(&fill_chars.foldclosed,  "foldclose"),
     CHARSTAB_ENTRY(&fill_chars.foldsep,	    "foldsep"),
+    CHARSTAB_ENTRY(&fill_chars.foldkeep,    "foldkeep"),
```

Comment:
It seems that `:set fillchars-=foldkeep:\ ` doesn't work. You need to add this below:
```diff
diff --git a/src/screen.c b/src/screen.c
index c2696cdc2..1216f8806 100644
--- a/src/screen.c
+++ b/src/screen.c
@@ -4859,6 +4859,7 @@ set_chars_option(win_T *wp, char_u *value, int is_listchars, int apply,
 		fill_chars.foldopen = '-';
 		fill_chars.foldclosed = '+';
 		fill_chars.foldsep = '|';
+		fill_chars.foldkeep = NUL;
 		fill_chars.diff = '-';
 		fill_chars.eob = '~';
 		fill_chars.lastline = '@';
```